### PR TITLE
Remove Eth2 Terminology from V2 Docs

### PR DESCRIPTION
**Pull request by rauljordan**
Changelog:
- [x] Remove the term eth2 and replace with Ethereum consensus or Ethereum proof-of-stake where appropriate
- [x] Remove some old pages such as spadina, zinken testnets
- [x] Remove ethdo from key management section as it is unused and not maintained

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -23,17 +23,11 @@ module.exports = {
                 {
                     type: 'docsVersionDropdown',
                     position: 'left',
-                    dropdownItemsAfter: [{ to: '/versions', label: 'All versions' }],
                     dropdownActiveClassDisabled: true,
                 },
                 {
                     to: 'docs/mainnet/joining-eth2',
```
Comment (james-prysm) [inline website/docusaurus.config.js:29]: does this need to be changed? docs/mainnet/joining-eth2

Comment (rauljordan) [reply website/docusaurus.config.js:29]: Yep, ill do that one in the next PR

File: website/versioned_docs/version-2.0.0/prysm-usage/web-interface.md [hunk 2]
```diff
@@ -55,7 +54,7 @@ The available parameters to customize are:
 If you are running your beacon node and validator on some server that you want to access from the outside, we recommend SSH local port forwarding to access it. For example, you would do the following from your home computer:
 
 ```
-ssh -L 7500:127.0.0.1:7500 user@host_ip
+sh -L 7500:127.0.0.1:7500 user@host_ip
```
Comment (james-prysm) [inline website/versioned_docs/version-2.0.0/prysm-usage/web-interface.md:57]: was this command supposed to be updated?

**Review by james-prysm — APPROVED** [review]
I think this looked good this time.

**Pull request merged**